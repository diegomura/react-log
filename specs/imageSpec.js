import React from 'react';
import render from './testRenderer';

describe('Image', () => {
  let output;

  test('Should render image with default styles', () => {
    output = render(
      <img src="http://lorempixel.com/400/200/sports/5/" width="400" height="200" />
    );
    expect(output).toMatchSnapshot();
  });

  test('Should render image with custom styles', () => {
    output = render(
      <img 
        src="http://lorempixel.com/400/200/sports/5/" 
        width="400" 
        height="200"
        style={{
          width: '800px',
          height: '400px',
          border: '2px solid red'
        }} />
    );
    expect(output).toMatchSnapshot();
  });
});
