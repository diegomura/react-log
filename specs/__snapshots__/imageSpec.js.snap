// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Should render image with custom styles 1`] = `
"
%c+
 background:url(http://lorempixel.com/400/200/sports/5/);background-size:800 400;color:transparent;font-size:1px;padding:200px 400px;line-height:400px;width:800px;height:400px;border:2px solid red;"
`;

exports[`Image Should render image with default styles 1`] = `
"
%c+
 background:url(http://lorempixel.com/400/200/sports/5/);background-size:400 200;color:transparent;font-size:1px;padding:100px 200px;line-height:200px;"
`;
